src/AppBundle/Controller/Api/AccountController.php

```diff
@@ -63,6 +64,24 @@ public function loginAction(Request $request)
         return new JsonResponse($userDto);
     }
 
+    /**
+     * @Route("api/account/logout")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
```

Comment:
```suggestion
     * @return Response
```
Krever `use Symfony\Component\HttpFoundation\Response;` på toppen av filen.
(PHPStorm fikser alt automatisk)